src/client/utils/Market.ts

```diff
@@ -2,13 +2,15 @@ import { MarketListing } from '../../interfaces/MarketListing';
 import CardUtils from './Card';
 
 import DateUtils from './Date';
+import PackUtils from './Pack';
```

Comment:
Please reorder properly the imports

Reply:
Sorry, I let my IDE do that for me!
What is the expected order? Grouped by business objects and then framework objects?

Reply:
There isn't a really given order at the moment, but I'll refactor it when I have some times to actually enforce a given order.